bin/src/hickory-dns.rs

```diff
@@ -44,6 +44,16 @@ use std::{
 };
 
 use clap::Parser;
+
```

Comment:
Nit: please drop the empty lines around the metrics-related imports, should be a part of the external crates import block.

Reply:
Done.